content/Watsonx/Assistant/102.md

```diff
@@ -10,12 +10,13 @@ updated: 2023-03-31
 
 <QuizAlert text='Heads up! Parts of this section will be on the quiz.'/>
 
-Learn about the fundamental building blocks of a virtual assistant.
+Learn about the fundamental building blocks of a virtual assistant and how to collaborate on an
+assistant with colleagues.
 
 ## Prerequisites
```

Comment:
At the end of the lab, Congratulate the user on the completion of the lab and provide the link for the next lab. You can look at the existing page for reference.